tests/snap_test.go

```diff
@@ -93,15 +92,16 @@ func setup() (teardown func(), err error) {
 }
 
 func setupGPIOMock(snapPath string) (string, error) {
-	if !useGPIOMock() || !utils.LocalServiceSnap() {
+	// Check if gpio mock is enabled and a local snap is provided
+	if !useGPIOMock() || (env.SnapPath() == "") {
 		return snapPath, nil
 	}
 
-	// check if gpio mock is enabled AND the service is running locally
 	// Run gpio mockup script
-	_, stderr, err := utils.Exec(nil, "./gpio-mock.sh")
+	stdout, stderr, err := utils.Exec(nil, "./gpio-mock.sh")
+	_ = utils.WriteLogFile(nil, "gpio-mock", stdout)
```

Comment:
Stdout alone may not be so useful. How about redirecting stderr to stdout first?

Reply:
I agree. At one point I wrote a second log file for stderr. I ended up removing it because the next lines will print stderr to the test's stdout if an error occurred. I however do like your suggestion. Let's do that and remove the print in the code.